common/scripted_triggers/missions_expanded_buildings_scripted_triggers.txt

```diff
@@ -262,10 +266,14 @@ province_has_building_of_group = {
 						check_variable = { equal_or_more_advanced_than_town_hall = 1 }
 					}
 				}
-				has_building = town_hall
+				OR = {
+					has_building = town_hall
+					has_building = government_lvl_3
+					has_building = government_lvl_4
+				}
 			}
 			else = {
-				has_building = university
+				has_building = government_lvl_4
```

Comment:
This is a part of 'Do I have a town_hall or better government building' clause. In my opinion it should not count Univerrsities in this chain. I applied the same logic as was applied below for other building chains.